junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/TestExecutionListenerRegistry.java

```diff
@@ -94,4 +109,12 @@ public void reportingEntryPublished(TestIdentifier testIdentifier, ReportEntry e
 
 	}
 
+	interface EagerTestExecutionListener extends TestExecutionListener {
```

Comment:
What are eager EagerTestExecutionListener? Shouldn't we add ordering instead?

Reply:
For now this is a one-off internal scenario. I found this easier to understand and less error-prone than adding ordering support.